src/BeadingStrategy/LimitedBeadingStrategy.cpp

```diff
@@ -31,6 +40,21 @@ LimitedBeadingStrategy::Beading LimitedBeadingStrategy::compute(coord_t thicknes
     {
         ret.toolpath_locations[bead_count - 1 - bead_idx] = thickness - ret.toolpath_locations[bead_idx];
     }
+
+    //Create a "fake" inner wall with 0 width to indicate the edge of the walled area.
+    //This wall can then be used by other structures to e.g. fill the infill area adjacent to the variable-width walls.
+    coord_t innermost_toolpath_location = ret.toolpath_locations[max_bead_count / 2 - 1];
```

Comment:
Is max_bead_count always even?
and what happens when max_bead_count = 1?

| max_bead_count | pos    |
|----------------|--------------|
| 1              | -1 |
| 2              | 0 |
| 3              | 0 |
| 4              | 1 |
| 5              | 1 |

Reply:
I think I don't need an answer anymore. Since max_bead_count is always multiplied by 2 when generating the wall_toolpaths